src/sagas.js

```diff
@@ -372,6 +376,11 @@ function* schedulerPerMinute (action) {
   }
 }
 
+function* fetchPermissionWithoutFilter (action) {
+  const data = yield callExternalAction('checkFilter', Api.checkFilter, { action: 'CHECK_FILTER' })
+  yield put(setUserFilterPermission(data.error !== undefined))
```

Comment:
@jelkosz , is this how the API is intended to be used for this kind of information?